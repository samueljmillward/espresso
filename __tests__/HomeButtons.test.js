import React from 'react';
import { shallow } from 'enzyme';
import HomeButtons from '../src/elements/HomeButtons';

describe('renders UI', () => {
  it('Navbar', () => {
    const wrapper = shallow(<HomeButtons />);
    expect(wrapper).toMatchSnapshot();
  });
});
