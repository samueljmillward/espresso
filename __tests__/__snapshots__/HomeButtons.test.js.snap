// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders UI Navbar 1`] = `
<div
  className="makeStyles-root-1"
>
  <WithStyles(ForwardRef(ButtonBase))
    className="makeStyles-image-2"
    component={
      Object {
        "$$typeof": Symbol(react.forward_ref),
        "propTypes": Object {
          "innerRef": [Function],
          "onClick": [Function],
          "replace": [Function],
          "target": [Function],
          "to": [Function],
        },
        "render": [Function],
      }
    }
    focusRipple={true}
    style={
      Object {
        "width": "50%",
      }
    }
    to="/beans"
  >
    <span
      className="makeStyles-imageSrc-4"
      style={
        Object {
          "backgroundImage": "url(/images/beans.jpg)",
        }
      }
    />
    <span
      className="makeStyles-imageBackdrop-5"
    />
    <span
      className="makeStyles-imageButton-3"
    >
      <WithStyles(ForwardRef(Typography))
        className="makeStyles-imageTitle-6"
        color="inherit"
        component="span"
        variant="subtitle1"
      >
        My Beans
      </WithStyles(ForwardRef(Typography))>
    </span>
  </WithStyles(ForwardRef(ButtonBase))>
  <WithStyles(ForwardRef(ButtonBase))
    className="makeStyles-image-2"
    component={
      Object {
        "$$typeof": Symbol(react.forward_ref),
        "propTypes": Object {
          "innerRef": [Function],
          "onClick": [Function],
          "replace": [Function],
          "target": [Function],
          "to": [Function],
        },
        "render": [Function],
      }
    }
    focusRipple={true}
    style={
      Object {
        "width": "50%",
      }
    }
    to="/brews"
  >
    <span
      className="makeStyles-imageSrc-4"
      style={
        Object {
          "backgroundImage": "url(/images/machine.jpg)",
        }
      }
    />
    <span
      className="makeStyles-imageBackdrop-5"
    />
    <span
      className="makeStyles-imageButton-3"
    >
      <WithStyles(ForwardRef(Typography))
        className="makeStyles-imageTitle-6"
        color="inherit"
        component="span"
        variant="subtitle1"
      >
        Pull A Shot
      </WithStyles(ForwardRef(Typography))>
    </span>
  </WithStyles(ForwardRef(ButtonBase))>
</div>
`;
